x/authority/client/cli/tx.go

```diff
@@ -171,3 +177,88 @@ For a 24-hour grace period the former authority key is equivalent to the new one
 	flags.AddTxFlagsToCmd(cmd)
 	return cmd
 }
+
+const (
+	UpgHeight = "upgrade-height"
+	UpgTime   = "upgrade-time"
```

Comment:
Guess we can remove this field and any uses of it.

Reply:
Yeah